README.md

```diff
@@ -1,26 +1,38 @@
-# Introduction
+<p align="right">
+  <a href="https://npmjs.org/package/react-map-gl">
+    <img src="https://img.shields.io/npm/v/react-map-gl.svg?style=flat-square" alt="version" />
+  </a>
+  <a href="https://travis-ci.org/uber/react-map-gl">
+    <img src="https://img.shields.io/travis/uber/react-map-gl/master.svg?style=flat-square" alt="build" />
+  </a>
+  <a href="https://npmjs.org/package/react-map-gl">
+    <img src="https://img.shields.io/npm/dm/react-map-gl.svg?style=flat-square" alt="downloads" />
+  </a>
+</p>
 
-react-map-gl is a suite of [React](http://facebook.github.io/react/) components for
-[Mapbox GL JS](https://github.com/mapbox/mapbox-gl-js), a WebGL-powered vector and raster tile mapping library. In addition to exposing `MapboxGL` functionality to React apps, react-map-gl also integrates seamlessly with [deck.gl](https://uber.github.io/deck.gl).
+<h1 align="center">react-map-gl | <a href="https://uber.github.io/react-map-gl">Docs</a></h1>
 
-# Installation
+<h5 align="center">
+React Components Suite for <a href="https://github.com/mapbox/mapbox-gl-js">Mapbox GL JS</a>.
+</h5>
 
-```sh
-npm install --save react-map-gl
-```
+### Introduction
+
+The premise of react-map-gl was to ease usage of both React and *MapboxGL*, the famous WebGL-powered vector and raster tile mapping library, while also allowing a seamless integration with [deck.gl](https://uber.github.io/deck.gl).
+
+### Installation
 
-## Using with Browserify, Webpack, and other JavaScript Bundlers
+    npm install --save react-map-gl
 
 * `browserify` - react-map-gl is extensively tested with `browserify` and works without configuration.
 
 * `webpack 2` - Most of the provided react-map-gl examples use webpack 2. For a minimal example, look at the [exhibit-webpack](https://github.com/uber/react-map-gl/tree/master/examples/exhibit-webpack) folder, demonstrating a working demo using `webpack 2`.
 
-* `create-react-app` - At this point configuration-free builds are not possible with webpack due to the way the mapbox-gl-js module is published. You will need to eject your app (sorry) and add one line alias to your webpack config.
+* `create-react-app` - At this point configuration-free builds are not possible with webpack due to the way the mapbox-gl-js module is published. You will need to eject your app the alias to your webpack config, as showed in the exhibit-webpack.
```

Comment:
Change last sentence to `You will need to add an alias to your webpack config, as shown in the exhibit-webpack`.